src/Raven.Server/Program.cs

```diff
@@ -187,9 +187,17 @@ public static int Main(string[] args)
                         }
                         catch (Exception e)
                         {
+                            var message =
```

Comment:
We need to detect that this was a port binding exception.

Reply:
We can detect this by checking the InnerException e.InnerException will be AddressInUseException

Reply:
On Linux as well?
What about not permission for low ports on Linux?